lib/Runtime/Library/TypedArray.cpp

```diff
@@ -2742,7 +2489,7 @@ namespace Js
             defaultConstructor = scriptContext->GetLibrary()->GetFloat32ArrayConstructor();
             break;
         case TypeId::TypeIds_Float64Array:
-            defaultConstructor = scriptContext->GetLibrary()->GetFloat32ArrayConstructor();
+            defaultConstructor = scriptContext->GetLibrary()->GetFloat64ArrayConstructor();
```

Comment:
Yes, look at the new test "prototype of the typedarray object should be same when species create invoked".

---
In reply to: [80845588](https://github.com/Microsoft/ChakraCore/pull/1629#discussion_r80845588) [](ancestors = 80845588)